File: U5/Java/U5P_1.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompan.u5p_1;
import java.util.Scanner;
/**
 *
 * @author alfre
 */
public class U5P_1 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Solicitar al usuario que ingrese dos números
        System.out.print("Ingrese el primer número: ");
        double num1 = scanner.nextDouble();

        System.out.print("Ingrese el segundo número: ");
        double num2 = scanner.nextDouble();

        // Mostrar el menú de operaciones
        System.out.println("\nSeleccione una operación:");
        System.out.println("1. Suma");
        System.out.println("2. Resta");
        System.out.println("3. Multiplicación");
        System.out.println("4. División");

        // Leer la opción del usuario
        System.out.print("Ingrese el número de la operación: ");
        int opcion = scanner.nextInt();

        // Realizar la operación seleccionada
        double resultado = 0;
        switch (opcion) {
            case 1:
                resultado = num1 + num2;
                break;
            case 2:
                resultado = num1 - num2;
                break;
            case 3:
                resultado = num1 * num2;
                break;
            case 4:
                // Verificar la división por cero
                if (num2 != 0) {
                    resultado = num1 / num2;
                } else {
                    System.out.println("Error: No se puede dividir por cero.");
                    return;
                }
                break;
            default:
                System.out.println("Opción no válida.");
                return;
        }

        // Mostrar el resultado
        System.out.println("El resultado es: " + resultado);
    }
}

File: U5/C#/U5P_1.cs
using System;

namespace U5P_1
{
    class U5P_1
    {
        static void Main(string[] args)
        {
            // Solicitar al usuario que ingrese dos números
            Console.Write("Ingrese el primer número: ");
            double num1 = Convert.ToDouble(Console.ReadLine());

            Console.Write("Ingrese el segundo número: ");
            double num2 = Convert.ToDouble(Console.ReadLine());

            // Mostrar el menú de operaciones
            Console.WriteLine("\nSeleccione una operación:");
            Console.WriteLine("1. Suma");
            Console.WriteLine("2. Resta");
            Console.WriteLine("3. Multiplicación");
            Console.WriteLine("4. División");

            // Leer la opción del usuario
            Console.Write("Ingrese el número de la operación: ");
            int opcion = Convert.ToInt32(Console.ReadLine());

            // Realizar la operación seleccionada
            double resultado = 0;
            switch (opcion)
            {
                case 1:
                    resultado = num1 + num2;
                    break;
                case 2:
                    resultado = num1 - num2;
                    break;
                case 3:
                    resultado = num1 * num2;
                    break;
                case 4:
                    // Verificar la división por cero
                    if (num2 != 0)
                    {
                        resultado = num1 / num2;
                    }
                    else
                    {
                        Console.WriteLine("Error: No se puede dividir por cero.");
                        return;
                    }
                    break;
                default:
                    Console.WriteLine("Opción no válida.");
                    return;
            }

            // Mostrar el resultado
            Console.WriteLine("El resultado es: " + resultado);
        }
    }
}
